apps/mobile/src/components/Feed/AdmireBottomSheet/AdmireBottomSheet.tsx

```diff
@@ -23,9 +26,10 @@ const SNAP_POINTS = [350];
 type AdmireBottomSheetProps = {
   feedEventId: string;
```

Comment:
this can be a post or feedEvent id right? if so lets rename to feedId for clarity - i think that matches CommentsBottomSheetprops too